### ISSUE TITLE
Light Estimation not working in released application

### ISSUE THREAD

**Issue by knightcube** [google-ar/arcore-unity-sdk#636]
### SPECIFIC ISSUE ENCOUNTERED
I developed a game in unity. Here is the closed beta version - https://play.google.com/store/apps/details?id=com.xrcubes.arminoes

When I developed and tested on my device the colors of the objects were like this - 
![Screenshot_20190902-090948](https://user-images.githubusercontent.com/22871961/64190268-96fa2100-ce93-11e9-8e1b-23901a5526da.png)

But on downloading from playstore, the colors are very dull - 
![WhatsApp Image 2019-09-03 at 20 41 13](https://user-images.githubusercontent.com/22871961/64190305-b5601c80-ce93-11e9-93e7-a83736a1a2db.jpeg)

What could be the potential reason for this?

### VERSIONS USED
- Unity: 2019.1.10f1
- ARCore SDK for Unity: v1.11.0
- Device manufacturer, model, and O/S:  Nokia, 8.0, Android 9.0

**Comment by bopangzz** [comment]
Can you clarify is the problem that the color looks different between development build and shipped build? Are you testing the app in the same environment? Not that if you have light estimation turned on, the lighting condition difference may affect the result.

**Comment by knightcube** [comment]
@bopangzz yes I am testing in the same environment and yes the problem is that color looks different in both the builds.

**Comment by knightcube** [comment]
So digged in a little more and found that if I keep the Player Settings > Other setting >Configuration > Scripting backend as Mono. Then the colors look bright. However the backend scipting needs to be IL2CPP so that the project supports ARM64 architecture which is, as of 1st August, a requirement for all apps on play store. 
Now what should I do fix this issue?

**Comment by bopangzz** [comment]
There is a bug in Unity 2019.1 and IL2CPP which is causing this issue: https://github.com/google-ar/arcore-unity-sdk/issues/614 
This should be fixed in next release.

**Comment by dax0229** [comment]
I temporarily fix it and applied it to my app, and it works fine. #627
